src/mesh/cartesiangrid.jl

```diff
@@ -53,28 +53,28 @@ julia> CartesianGrid((-1.,),(1.,), dims=(100,))
 struct CartesianGrid{Dim,T} <: Mesh{Dim,T}
   dims::Dims{Dim}
   origin::Point{Dim,T}
-  spacing::SVector{Dim,T}
+  spacing::Vec{Dim,T}
```

Comment:
You can leave this as SVector

Reply:
I made this property a Vec because CartesianGrid uses operations between Vec and Points in its methods.